plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/S3SelectLineRecordReader.java

```diff
@@ -212,6 +213,10 @@ private int readLine(Text value)
                     });
         }
         catch (Exception e) {
+            if (e instanceof InterruptedException) {
```

Comment:
> Reset interrupted flag


"Reset interrupted flag in S3 Select reader"

Reply:
Same thing here about the SDK client throwing `AbortedException` when it encounters an `InterruptedException` internally.